### DISCUSSION TITLE
Audio book & ebook requestor

### DISCUSSION THREAD

**Discussion by jw2804** [activexray/Doplarr#129] [Ideas]
Is this still being maintained ? i`m not seeing much development. 

It works great for movies and series but wondering if ebooks and audio books could be added ? ie:

/movies
/series
/ebooks
/audionbooks

Thanks

**Comment by activexray** [comment]
It is maintained as in I’ll fix reproducible bugs and breaking changes in the discord api. I currently don’t have the time or interest to add new features unfortunately.

**Reply by jw2804** [reply]
No worries, and many thanks for this great requestor app.
